.github/workflows/container-tests.yml

```diff
@@ -84,6 +84,14 @@ jobs:
         with:
           ref: "refs/pull/${{ github.event.issue.number }}/head"
 
+      - name: Obtain envvars from PR's comment
+        id: user_defined
+        shell: bash
+        run : |
+          _comment=${{ github.event.comment.body }}
+          _vars=$(echo $_comment | grep -Eo '\[ENV:.*\]' | cut -d':' -f2 | tr -d ']')
```

Comment:
I would also check if variable [ENV:SOME_KEY:SOME_VALUE] contains both SOME_KEY and SOME_VALUE.

Reply:
Is this really needed?
Only members or maintainers of the sclorg would be able to use it anyway. I personally would add this to documentation somewhere, so these people know how to use it.
But if you insist I will rework this.

Reply:
I am thinking if the user can specify a variable that can run some weird code or even if it can damage infrastructure.
The tests can be triggered by owners or maintainers of our repositories, so may be I am too paranoid :)

Reply:
Hmm, I need to think about it.

Reply:
No Python please. This is ok.